ProjectLighthouse/Controllers/UserController.cs

```diff
@@ -24,18 +25,32 @@ public UserController(Database database)
             this.database = database;
         }
 
-        [HttpGet("user/{username}")]
-        public async Task<IActionResult> GetUser(string username)
+        public async Task<string> GetSerializedUser(string username)
         {
-            User user = await this.database.Users.Include(u => u.Location).FirstOrDefaultAsync(u => u.Username == username);
+            User? user = await this.database.Users.Include(u => u.Location).FirstOrDefaultAsync(u => u.Username == username);
+            return user?.Serialize();
+        }
 
+        [HttpGet("user/{username}")]
+        public async Task<IActionResult> GetUser(string username) {
+            string? user = await this.GetSerializedUser(username);
             if (user == null) return this.NotFound();
-
-            return this.Ok(user.Serialize());
+            return this.Ok(user);
         }
 
         [HttpGet("users")]
-        public async Task<IActionResult> GetUserAlt([FromQuery] string u) => await this.GetUser(u);
+        public async Task<IActionResult> GetUserAlt([FromQuery] string[] u)
+        {
+            List<string> tasks = new();
```

Comment:
`tasks`?

Reply:
Old variable name, my previous strategy was running async calls in parallel but I learned you can't access a single DbContext from multiple threads at the same time. The choices were to have a context for each async call or make it synchronous, so I went with synchronous and forgot to update the name to indicate I wasn't getting a list of tasks anymore. Updating this for clarity in my next commit.

Reply:
Fixed in https://github.com/LBPUnion/project-lighthouse/pull/38/commits/45b8464d1cb676080d24e3cc5a4036ff65d804cb